.github/workflows/instrumentations_1.yml

```diff
@@ -37,8 +38,6 @@ jobs:
           - "resource-detector-container"
         os: [ubuntu-20.04]
         exclude:
-          - python-version: py311
-            package: "prometheus-remote-write"
```

Comment:
Why remove this but not replace it with 3.12?

Reply:
Because tests works fine with 3.11 and 3.12, so no need to exclude them